x/auth/tx/legacy_amino_json.go

```diff
@@ -20,6 +20,10 @@ var _ signing.SignModeHandler = signModeLegacyAminoJSONHandler{}
 // SignModeHandler.
 type signModeLegacyAminoJSONHandler struct{}
 
+func NewSignModeLegacyAminoJSONHandler() signing.SignModeHandler {
```

Comment:
This function is added to support tests in `/integration`.

Reply:
nit: could we get a godoc saying that.